### PR TITLE
Add new start page and contact CICA page

### PR DESCRIPTION
**Pull request by BarryPiccinni**
Add new application start page and new contact CICA page.

Manual test pass
Lint pass
UAT signed off

### CODE REVIEW

File: src/template.js [hunk 1]
```diff
@@ -339,10 +341,12 @@ module.exports = {
...
         'p-rep-telephone-number': repTelephoneNumber.section,
         'p-rep-reference-number': repReferenceNumber.section,
-        'p-rep-claims-management-reg': repClaimsManagementRegNumber.section
+        'p-rep-claims-management-reg': repClaimsManagementRegNumber.section,
+        'p--new-or-existing-application': newOrExistingApp.section,
+        'p--contact-cica': contactCica.section
     },
     routes: {
-        initial: 'p-applicant-fatal-claim',
+        initial: 'p--new-or-existing-application',
```
Comment (neil-stephen-mcgonigle) [inline src/template.js:349]: change this back to p-applicant-fatal-claim

File: src/template.js [hunk 2]
```diff
@@ -530,11 +534,13 @@ module.exports = {
...
             'p-rep-reference-number': repReferenceNumber.route,
-            'p-rep-claims-management-reg': repClaimsManagementRegNumber.route
+            'p-rep-claims-management-reg': repClaimsManagementRegNumber.route,
+            'p--new-or-existing-application': newOrExistingApp.route,
+            'p--contact-cica': contactCica.route
         }
     },
     answers: {},
-    progress: ['p-applicant-fatal-claim'],
+    progress: ['p--new-or-existing-application'],
```
Comment (neil-stephen-mcgonigle) [inline src/template.js:543]: and here change to p-applicant-fatal-claim

**Review by neil-stephen-mcgonigle — CHANGES_REQUESTED** [review]

File: src/template.js (hunk 1 above)
Comment (BarryPiccinni) [reply src/template.js:349]: This is the new first page of the application, so it should replace `p-applicant-fatal-claim` as the initial page.

File: src/template.js (hunk 2 above)
Comment (BarryPiccinni) [reply src/template.js:543]: Same as above, this is now the first page in the application, so it needs to replace 'p-applicant-fatal-claim' here.

Comment (neil-stephen-mcgonigle) [reply src/template.js:543]: ah of course¬

**Pull request merged**